src/main/resources/view/MainWindow.fxml

```diff
@@ -25,36 +27,47 @@
 
       <VBox>
         <MenuBar fx:id="menuBar" VBox.vgrow="NEVER">
-          <Menu mnemonicParsing="false" text="File">
-            <MenuItem mnemonicParsing="false" onAction="#handleExit" text="Exit" />
+          <Menu fx:id="menuHelpTab" mnemonicParsing="false" text="Help">
+            <MenuItem fx:id="helpMenuItem" styleClass="menu-item" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+          </Menu>
+          <Menu fx:id="menuOverviewTab" mnemonicParsing="false" text="Overview">
+            <MenuItem fx:id="overviewTabItem" styleClass="menu-item"  mnemonicParsing="false" text="Display Overview" onAction="#handleOverview"/>
+          </Menu>
+          <Menu fx:id="menuIncomeTab" mnemonicParsing="false" text="Income">
+            <MenuItem fx:id="incomeTabItem" styleClass="menu-item" mnemonicParsing="false" text="Display Income" onAction="#handleIncome"/>
+          </Menu>
+          <Menu fx:id="menuExpenseTab" mnemonicParsing="false" text="Expenses">
+            <MenuItem fx:id="expenseTabItem" styleClass="menu-item"  mnemonicParsing="false" text="Display Expense" onAction="#handleExpense"/>
           </Menu>
-          <Menu mnemonicParsing="false" text="Help">
-            <MenuItem fx:id="helpMenuItem" mnemonicParsing="false" onAction="#handleHelp" text="Help" />
+          <Menu fx:id="menuAnalyticsTab" mnemonicParsing="false" text="Analytics">
+            <MenuItem fx:id="analyticsTabItem" styleClass="menu-item"  mnemonicParsing="false" text="Display Analytics" onAction="#handleAnalytics"/>
           </Menu>
```

Comment:
Should buttons be used instead of menu? Having to click the `Menu`, then the `MenuItem` for tabs is very clunky.

![image](https://user-images.githubusercontent.com/5585517/95324635-9b80dc80-08d2-11eb-83c9-c788b1d9ce77.png)

Reply:
@siddarth2824 will be addressing this in a separate PR.